### DISCUSSION TITLE
如何直接在HTML中使用ANTD，并且支持IE11

### DISCUSSION THREAD

**Discussion by gvcode1** [ant-design/ant-design#35184] [Q&A]
Hi大家好，目前项目中有个BT的需求，就是必须得在本地HTML中使用ANTD，这个网上已经有解决方案。但是这样的情况下，还需要支持IE11，目前实在找不到方案，引入react-app-polyfill的话，就不能再本地HTML中来直接使用，如何手动引入本地的polyfill呢？（主要不知道最终是哪些polyfill文件需要引入）

谢谢大家帮忙了

**Comment by yoyo837** [comment]
babel polyfill有静态cdn

**Comment by gvcode1** [comment]
非常感谢，引入了polyfill后，IE11工作正常
